docs/SDL/GetListOfPermissions/index.md

```diff
@@ -85,39 +85,34 @@ Possible Layout - ExternalConsentStatus
 
 ```json
 {
-  "id" : 143,
-  "jsonrpc" : "2.0",
-  "result" :
-  {
-    "allowedFunctions" :
-
-[
-      {
-    "name" : "Location-1",
-      "id":1234,
-      "allowed":true
-  },
-
-     {
-    "name" : "Notifications",
-      "id":76876,
-      "allowed":false
-  },
-
-    ]
-      "externalConsentStatus":
-[       
-       {
-       “entityType” : “0”,
-       “entityID”   : “126”,
-       “status” : “ON”
-       }
-]
-
-
-    "code" : 0,
-    "method" : "SDL.GetListOfPermissions"
-  }
+	"id": 143,
+	"jsonrpc": "2.0",
+	"result": {
+		"allowedFunctions":
+
+			[{
+					"name": "Location-1",
+					"id": 1234,
+					"allowed": true
+				},
+
+				{
+					"name": "Notifications",
+					"id": 76876,
+					"allowed": false
+				}
+
+			],
+		"externalConsentStatus": [{
+			"entityType": "0",
+			"entityID": "126",
```

Comment:
`entityType` and `entityID` are integers and shouldn't be in quotes

Reply:
Resolved in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/155/commits/f84f198b7223d6940c1d2103ca3965f8d29ed8a4